.circleci/config.yml

```diff
@@ -179,6 +193,14 @@ workflows:
           filters:
             tags:
               only: /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
+      - scripted3_jdk:
+          requires:
+            - lint
+          filters:
+            branches:
+              only: series/2.x
```

Comment:
I think you need to remove this filter. Because of it, it's not running on your PR and won't be on other PRs. The 1.x branch won't have this change in ci config so it won't run it anyway.

Reply:
Done